list.yaml

```diff
@@ -474,3 +474,10 @@
 
 - title: barkbox for cats, but its just a box
   desc: kitnipbox already exists, but instead of shipping a bunch of actual cat stuff, we just ship a box to you every 2 weeks for a subscription fee.
+
+- title: bars
+  desc: App that lets you search for specific rap verses with timestamps.
```

Comment:
The latter.  Basically what Genius does except add timestamps for when each verse starts and ends.